docs/en_US/NAS/retiarii/Tutorial.rst

```diff
@@ -182,13 +182,19 @@ Here is a simple example of using trainer and strategy.
 
 .. code-block:: python
 
-  trainer = PyTorchImageClassificationTrainer(base_model,   
-    dataset_cls="MNIST",
-    dataset_kwargs={"root": "data/mnist", "download": True},
-    dataloader_kwargs={"batch_size": 32},
-    optimizer_kwargs={"lr": 1e-3},
-    trainer_kwargs={"max_epochs": 1})
-  simple_startegy = RandomStrategy()
+  import nni.retiarii.trainer.pytorch.lightning as pl
+  from nni.retiarii import blackbox
+  from torchvision import transforms
+
+  transform = transforms.Compose([transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))])
+  train_dataset = blackbox(MNIST, root='data/mnist', train=True, download=True, transform=transform)
```

Comment:
`blackbox` is strange here, because i don't understand why it creates `train_dataset`

Reply:
o... `MNIST` is a class name?

Reply:
Yes. This is limited by serialization.

Reply:
is it possible we also wrap the dataset class by default? when users want to define their own dataset class, they decorate this class with for example `@register_dataset`

Reply:
another option, for this pr, is renaming `blackbox` to `make_serializable`